auth/auth.config.js

```diff
@@ -0,0 +1,30 @@
+const LocalStrategy = require("passport-local");
+const logger = require("../logs/logger");
+const db = require("../models");
+
+const User = db.users;
+
+module.exports = (passport) => {
+  passport.use(
+    "local-login",
+    new LocalStrategy(
+      { usernameField: "email", passwordField: "password" },
+      (email, password, done) => {
+        User.findOne({ where: { email: email } })
+          .then((user) => {
+            if (!user) {
+              return done(null, false, { message: "User not found" });
+            }
+            if (user.password !== password) {
```

Comment:
Correct me if I am wrong, based on this check, it seems you currently store plain password in your database. if so, checkout libraries for hashing password like Bcrypt password encoder. For instance this libraray https://www.npmjs.com/package/bcrypt.

Password should never be stored in plain text. It must be hashed such that nobody can retrieve the plain text from it. (Note, it should not be encrypted because encrypted text can be decrypted once anybody has access to the secret/private key as the case may be)